app/shared/models.py

```diff
@@ -2,10 +2,10 @@
 
 from django.db import models
 
+
 class CommonInfo(models.Model):
```

Comment:
Like @andela-uawili pointed out, I'm of the opinion of making `CommonInfo` a mixin. Though not likely, considering the simplicity of what we are building, we get to delegate several specific properties and behaviors to several mixins which more-or-less is in line with the ISP concept (moreover python does not have interfaces).
@andela-ooshodi, the side-effect of using just one common class which some others would extend would be become apparent when 2 or more classes have some other fields different from `time_created` and `time_modified` in common. To use composition, we would have to instantiate an object of the class if we want to access its method and instance variables all the time in `__init__`. I know we don't need to do that in the case of trying to access class variables like `time_created` and `time_modified`.
Anyway, @andela-kanyanwu I don't think this class needs any form of timestamp. Practically in my own opinion, it's of little or no benefit indicating the time a weekday was created.